amt.go

```diff
@@ -295,6 +306,50 @@ func (n *Node) forEachAt(ctx context.Context, bs cbor.IpldStore, height int, sta
 
 }
 
+func (n *Node) parForEachAt(ctx context.Context, bs cbor.IpldStore, height int, start, offset uint64, parNum int, cb func(uint64, *cbg.Deferred) error) error {
+	errCh := make(chan struct{})
+	toProcess := make(chan struct{}, parNum)
+	for i := 0; i < parNum; i++ {
+		toProcess <- struct{}{}
+	}
+
+	var wg sync.WaitGroup
+	multiErr := &MultiError{}
+	wrapCh := func(u uint64, deferred *cbg.Deferred) error {
+		select {
+		case <-errCh:
+			return errors.New("just for stop")
+		default:
+		}
+		wg.Add(1)
+		select {
+		case <-toProcess:
+			go func() {
+				defer func() {
+					wg.Done()
+				}()
```

Comment:
This can be just `defer wg.Done()` -- there's no need for the anonymous function that's wrapping it.